rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1518,7 +1518,7 @@ void ppu_load_exec(const ppu_exec_object& elf)
 	p.stack_addr = vm::cast(vm::alloc(primary_stacksize, vm::stack, 4096));
 	p.stack_size = primary_stacksize;
 
-	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] Thread (main_thread)", p, "main_thread", primary_prio, 1);
+	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] main_thread ", p, "main_thread", primary_prio, 1);
```

Comment:
Was the space intentional?

```suggestion
	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] main_thread", p, "main_thread", primary_prio, 1);
```